src/python_testing/TC_ACL_2_11.py

```diff
@@ -118,16 +118,16 @@ async def test_TC_ACL_2_11(self):
                 restriction_type = restriction.type
                 ID1 = restriction.id
 
-                attribute = ALL_ATTRIBUTES[C1][ID1]
-                command = ALL_ACCEPTED_COMMANDS[C1][ID1]
-
                 if restriction_type == AccessControl.Enums.AccessRestrictionTypeEnum.kAttributeAccessForbidden:
+                    attribute = ALL_ATTRIBUTES[C1][ID1]
```

Comment:
How does this handle wildcards? Was this tested against wildcards?

Reply:
fixed.